tools/tpm2_createprimary.c

```diff
@@ -160,7 +159,7 @@ int createPrimary(TSS2_SYS_CONTEXT *sysContext, TPMI_RH_HIERARCHY hierarchy,
     inSensitive->t.sensitive.data.t.size = 0;
     inSensitive->t.size = inSensitive->t.sensitive.userAuth.b.size + 2;
 
-    if(setAlg(type, nameAlg, inPublic))
+    if(setAlg(type, nameAlg, inPublic, E_flag))
```

Comment:
E_flag, when passing this into routines, change it to is_policy_enforced, so it has more meaning.